cves/kernel/CVE-2018-20961.yml

```diff
@@ -173,8 +175,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: no tools/tests mentioned in commits
```

Comment:
It mentions MOXCAFE. I see little info on it, but it should be mentioned as it _is_ in the fix commit